src/main/java/com/github/redouane59/twitter/helpers/AbstractRequestHelper.java

```diff
@@ -29,19 +26,21 @@
 
   private final TwitterCredentials twitterCredentials;
   private final OAuth10aService service;
+  protected IAPIEventListener listener = null;
```

Comment:
Is it still needed there if now it passes trough the method arguments ?

Reply:
Forgot it. Fixing.